api-module-library/google-drive/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@friggframework/api-module-google-drive",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
You can remove this version bump as it will be done automatically through the `patch` label.